src/pages/workspace/WorkspacePageWithSections.js

```diff
@@ -88,7 +88,9 @@ class WorkspacePageWithSections extends React.Component {
 
     render() {
         const achState = lodashGet(this.props.reimbursementAccount, 'achData.state', '');
-        const hasVBA = achState === BankAccount.STATE.OPEN;
+        const hasVBA = this.props.reimbursementAccount.loading
+            ? null
+            : achState === BankAccount.STATE.OPEN;
```

Comment:
**suggestion:** something like `hasVBA` and `isLoadingVBA` could be used together in these components to have logic like:

```js
!isLoadingVBA && hasVBA -> show this
!isLoadingVBA && !hasVBA -> show that
```

Was just thinking someone might see `this.props.hasVBA === false` and refactor it to `!this.props.hasVBA` and  might re-introduce the bug.

Reply:
Yeah, very good point! I'll get a PR up for that change. Thanks!

Reply:
Nice catch Marc!